Commencement.Core/Services/DbService.cs

```diff
@@ -0,0 +1,28 @@
+﻿using System.Configuration;
+using System.Data.Common;
+using System.Data.SqlClient;
+
+namespace Commencement.Core.Services
+{
+    public interface IDbService
+    {
+        DbConnection GetConnection(string connectionString = null);
+    }
+
+    public class DbService : IDbService
+    {
+        public DbConnection GetConnection(string connectionString = null)
+        {
+            //If connection string is null, use the default sql ce connection
+            if (connectionString == null)
+            {
+                connectionString = ConfigurationManager.ConnectionStrings["MainDB"].ConnectionString;
+            }
+
+            var connection = new SqlConnection(connectionString);

```

Comment:
@jSylvestre We should try to use the reliable SQL connection from P&P like we do for give.  Might even want to think about it for the main nhibernate connection but that can be later.